### ISSUE TITLE
Understanding Guideline 2.3: Seizures and Physical Reactions

### ISSUE THREAD

**Issue by caztcha** [w3c/wcag#4328]
Understanding Guideline 2.3: Seizures and Physical Reactions | WAI | W3C
https://www.w3.org/WAI/WCAG22/Understanding/seizures-and-physical-reactions

This page contains the description "WCAG 2.0" in the "Intent" section. Because the 2.0 is outdated currently, shouldn't it be "WCAG 2" instead?

> The objective of this guideline is to ensure that content that is marked as conforming to WCAG 2.0 avoids the types of flash that are most likely to cause seizure when viewed even for a second or two.

Kind regards,

**Comment by patrickhlauke** [comment]
I *think* we held off updating every mention of 2.0 in https://github.com/w3c/wcag/pull/3588 because of the mention in https://www.w3.org/TR/WCAG22/#background-on-wcag-2

> Where this document refers to WCAG 2 it is intended to mean any and all versions of WCAG that start with 2.

Happy to reconsider though - there are quite a few other mentions of 2.0 in various understanding docs beyond that one.

**Comment by stevefaulkner** [comment]
> I think we held off updating every mention of 2.0 in https://github.com/w3c/wcag/pull/3588 because of the mention in https://www.w3.org/TR/WCAG22/#background-on-wcag-2

Isn't that a reason to change it to "WCAG 2" rather than specifically refer to an outdated version?

**Comment by bruce-usab** [comment]
I suspect that Understanding for Guidelines (as opposed to Success Criteria) have fallen through the cracks.

When someone volunteers for this, please also delete:
> Ensuring that content does not violate spatial pattern thresholds (future link)

**Comment by scottaohara** [comment]
i made pr #4361 for this and have added it to the drafted column in the wcag 2.x project.  so removing this issue from the project.
